WordPress/Classes/Models/QuickStartTourState.swift

```diff
@@ -1,3 +1,6 @@
+import CoreData
+
+@available(*, deprecated, message: "No longer used")
```

Comment:
`obsolete` is probably more appropriate than `deprecated` here.

Reply:
I'll remove these in the upcoming PRs.